Android/ThirdPartyAuth/app/src/main/java/com/bentley/sample/thirdpartyauth/LoginViewModel.kt

```diff
@@ -0,0 +1,55 @@
+package com.bentley.sample.thirdpartyauth
+
+import android.content.Context
+import android.content.Intent
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.auth0.android.Auth0
+import com.auth0.android.authentication.AuthenticationException
+import com.auth0.android.callback.Callback
+import com.auth0.android.provider.WebAuthProvider
+import com.auth0.android.result.Credentials
+import com.bentley.sample.shared.MainActivity
+
+class LoginViewModel(
+	resourceHelper: IResourceHelper,
+	onAuth0TokenUpdated: (auth0Token: String)->Unit = {},
```

Comment:
The rest of our code (for example ITMMessenger.kt) puts spaces around the arrow in declarations like this:

```kotlin
        onAuth0TokenUpdated: (auth0Token: String) -> Unit = {},
```
The Kotlin docs surround the arrows with a space on either side, so we should do the same.